src/utils.ts

```diff
@@ -30,6 +30,30 @@ class Utils {
     );
   }
 
+  /**
+   * Validate if the given string is a valid X-address on the XRP Ledger.
+   *
+   * @see https://xrpaddress.info/
+   *
+   * @param address An address to check.
+   * @returns True if the address is a valid X-address, otherwise false.
+   */
+  public static isValidXAddress(address: string): string {
+    return addressCodec.isValidXAddress(address);
+  }
+
+  /**
+   * Validate if the given string is a valid classic address on the XRP Ledger.
```

Comment:
`Validate whether the given string is a valid classic address for the XRP Ledger.`